x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/physical/FieldExtractExec.java

```diff
@@ -13,17 +13,25 @@
 import org.elasticsearch.xpack.ql.tree.Source;
 
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
 import java.util.Objects;
+import java.util.Set;
 
 public class FieldExtractExec extends UnaryExec implements EstimatesRowSize {
     private final List<Attribute> attributesToExtract;
     private final Attribute sourceAttribute;
+    private final Set<Attribute> forStats;
 
-    public FieldExtractExec(Source source, PhysicalPlan child, List<Attribute> attributesToExtract) {
```

Comment:
I'm not sure what that has top do with compatibility, but I've re-ordered the constructors, if that was what you were asking for.

Reply:
```java
 public FieldExtractExec(Source source, PhysicalPlan child, List<Attribute> attributesToExtract) {
    this(source, child, attributesToExtract, emptySet())
 }
```

It might be you already did that and I missed you moving the constructor around :)